packages/ui-icons/docs/keywords.json

```diff
@@ -9621,427 +10875,506 @@
     "category": "GIS",
-    "release": "4.2.0"
+    "release": "4.2.0",
+    "codepoint": ""
   },
   "spike-data-visualization-outlined": {
     "alias": ["4.2.0", "generic", "display", "raster", "interactive", "data"],
     "category": "Generic",
-    "release": "4.2.0"
+    "release": "4.2.0",
+    "codepoint": ""
   },
   "calendar-heat-chart": {
     "alias": ["4.2.0", "gis", "data", "display", "calendar", "value", "patterns"],
     "category": "GIS",
-    "release": "4.2.0"
+    "release": "4.2.0",
+    "codepoint": ""
   },
   "matrix-heat-chart": {
     "alias": ["4.2.0", "gis", "analysis", "categorical", "count", "variable"],
     "category": "GIS",
-    "release": "4.2.0"
+    "release": "4.2.0",
+    "codepoint": ""
   },
   "progress-bar-chart": {
     "alias": ["4.2.0", "charts and graphs", "indicator", "input", "control", "adjustment"],
     "category": "Charts and Graphs",
-    "release": "4.2.0"
+    "release": "4.2.0",
+    "codepoint": ""
   },
   "spike-data-visualization": {
     "alias": ["4.2.0", "generic", "display", "raster", "interactive", "data"],
     "category": "Generic",
-    "release": "4.2.0"
+    "release": "4.2.0",
+    "codepoint": 61758
   },
   "spike-data-visualization-open": {
     "alias": ["4.2.0", "generic", "display", "raster", "interactive", "data", "open", "shape"],
     "category": "Generic",
-    "release": "4.2.0"
+    "release": "4.2.0",
+    "codepoint": 61761
   },
   "spike-data-visualization-outlined-color-locked": {
     "alias": ["4.2.0", "generic", "display", "raster", "interactive", "data", "outline", "shape"],
     "category": "Generic",
-    "release": "4.2.0"
+    "release": "4.2.0",
+    "codepoint": ""
   },
   "spike-data-visualization-outlined-open": {
-    "alias": ["4.2.0", "generic", "display", "raster", "interactive", "data", "outlined", "shape", "open"],
+    "alias": ["4.2.1", "generic", "display", "raster", "data", "interactive"],
     "category": "Generic",
-    "release": "4.2.0"
+    "release": "4.2.1",
+    "codepoint": 61759
   },
   "spike-data-visualization-outlined-open-color-locked": {
     "alias": ["4.2.0", "generic", "display", "raster", "interactive", "data", "shape", "color"],
     "category": "Generic",
-    "release": "4.2.0"
+    "release": "4.2.0",
+    "codepoint": ""
   },
   "transfer": {
     "alias": ["4.2.1", "people", "move", "shift", "remove", "conduct", "transmit"],
     "category": "People",
-    "release": "4.2.1"
+    "release": "4.2.1",
+    "codepoint": 61762
   },
   "tapestry": {
     "alias": ["4.2.1", "brand", "segmentation", "data", "classification", "group"],
     "category": "Brand",
-    "release": "4.2.1"
+    "release": "4.2.1",
+    "codepoint": 61763
   },
   "spike-data-visualization-outlined-gradient": {
     "alias": ["4.2.1", "generic", "display", "raster", "interactive", "data"],
     "category": "Generic",
-    "release": "4.2.1"
-  },
-  "spike-data-visualization-outlined-open": {
```

Comment:
Looks like this item got merged with the previous `"spike-data-visualization-outlined-open"`.